### ISSUE TITLE
Can't upload CSV : POST on /dashboard/ fails -> 404 Not Found

### ISSUE THREAD

**Issue by lorisdev** [nocodb/nocodb#3542]
```
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **true**
Database: **sqlite3**
ProjectOnRootDB: **true**
RootDB: **sqlite3**
PackageVersion: **0.96.1**
```

**Steps To Reproduce**

1. Go to `Add new table` -> `Quick Import from` -> `CSV File`
2. Add a .csv file in the file input
3. Click on the button `Import`

**Expected Behavior**

The file should upload and then a Table should be created from it

**Actual Behavior**

An error occurs, when I check the network there is a POST request on `http://localhost:8080/dashboard/` with the file as payload and the response code is `404 Not Found`

**Comment by o1lab** [comment]
Pls help us with some logs in server and browser.

**Comment by dstala** [comment]
Issue reproducible in docker setup.

**Comment by dstala** [comment]
@lorisdev 
We are working to fix the error notification. Meanwhile, we did notice that - the import option gets enabled in spite of the error notification & it works. You can ignore the error for now & proceed further to create the table.

**Comment by pranavxc** [comment]
Fixed in `0.96.2`
